pkg/apis/node/v1alpha1/conversion.go

```diff
@@ -33,23 +33,37 @@ func addConversionFuncs(s *runtime.Scheme) error {
 func Convert_v1alpha1_RuntimeClass_To_node_RuntimeClass(in *v1alpha1.RuntimeClass, out *node.RuntimeClass, s conversion.Scope) error {
```

Comment:
Conversion functions  generally call the `autoConvert` function and then fixup the interesting places.  In this cases it must be totally manual because of `spec`.  Can we leave a comment at the top of these functions about why?